src/root.component.tsx

```diff
@@ -46,6 +69,14 @@ function Root(props) {
             </div>
           </div>
         </main>
+        <ContextWorkspace
+          title={workspaceTitle}
+          extensionSlot={currentWorkspaceExtensionSlot}
+          clearExtensionSlot={() => {
```

Comment:
We could place this in a `useCallback` hook to avoid respecifying it per render.